### DISCUSSION TITLE
<DynaSim>

### DISCUSSION THREAD

**Discussion by caiocaraujojunior** [community/community#129172] [Programming Help]
### Select Topic Area

Question

### Body

Hi, I am Caio and I am starting to work with DynaSim. I don't have programming skills.
I am also trying to find out how to use the GitHub tools. I am a psychiatrist interested in neural mass models, but I am very naive in this wonderful area.

The problem is that I receive an error message even after copying and pasting the exact 'scrip' from the dynaSim tutorial. For example, to implement the Hodgkin-Huxley Neuron.

Could anybody help me with these kinds of problems, please?

Thank you very much.

It will be a pleasure to keep in contact with all of you.

Caio

**Comment by ebndev** [comment]
Hi @caiocaraujojunior, welcome the GitHub Community!

If you still need support, please share links to what you're referring to and/or if you have made any progress. Providing more details might help someone give you a nudge in the right direction 😄
